config/initializers/county_to_region_code.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 # Taken from https://gist.github.com/radiac/d91d2ed1b971c03d49e9b7bd85e23f1c
+
+# rubocop:disable Metrics/CollectionLiteralLength
```

Comment:
Can we just exclude the initialisers directory instead?

Reply:
I originally excluded the file, but we decided it might be better to do it this way so we don't keep adding more tech debt to this file

Reply:
It's likely we're going to be violating this rule in this directory as we may need to setup a lot of configuration based stuff (eg enum lists to be re-used etc). Let's keep it for now but we should remove it if we need to add another block elsewhere (in the same directory)

Reply:
Yup agree, we should remove it if we have to add another one. I was hoping we could somehow abide by the rule going forward.